### ISSUE TITLE
[PNI Privacy Index] Implement animated eyes

### ISSUE THREAD

**Issue by mmmavis** [MozillaFoundation/foundation.mozilla.org#11153]
## Dev Tasks

Implement animated eyes
- [x] eyes of the emoji faces should follow cursor (or tap on touch screens)

**Comment by mmmavis** [comment]
Turns out it's a lot complicated to use JS to manipulate our emoji SVGs due to their structures and inner nodes. The proof-of-concept test I did a while back was to use JS to manipulate HTML elements (eg., `div` along with CSS for styling) to achieve the animated effect. That implementation was pretty straight-forward so I think animating SVG is most likely out of the question here.

Since now we are running short on dev time, I would like to propose a few options

1. Don't have these animated eyes **at all**. Replace them with static images instead.
2. Don't have these animated eyes and replace them with static images **for now** . Post-MVP we can add the animation by drawing pupils using HTML/JS/CSS, laying them on top of the static images so they are correctly positioned in the eyes, and using JS to make them follow cursor.
